src/main/java/uk/gov/hmcts/reform/pip/account/management/model/AzureAccount.java

```diff
@@ -5,43 +5,49 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import uk.gov.hmcts.reform.pip.account.management.validation.annotations.ValidEmail;
-import uk.gov.hmcts.reform.pip.account.management.validation.annotations.ValidName;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
 
 /**
- * Model that represents a subscriber.
+ * Model that represents an Azure Account.
  */
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-@ValidName
-public class Subscriber {
+public class AzureAccount {
 
     /**
      * The is the object ID that is returned from azure.
      */
     @ApiModelProperty(hidden = true)
-    private String azureSubscriberId;
+    private String azureAccountId;
 
     /**
-     * The email address for the subscriber.
+     * The email address for the account.
      */
     @ValidEmail
     private String email;
 
     /**
-     * The title of the subscriber.
+     * The first name of the account.
      */
-    private String title;
+    @NotEmpty
```

Comment:
should we get rid of this for the requirement that media members dont need to provide their name on migration?